<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapWidgetSettings</class>
 <widget class="QWidget" name="MapWidgetSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>268</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Map Settings</string>
  </property>
  <property name="locale">
   <locale language="C" country="AnyCountry"/>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="10" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QToolButton" name="btnMapFile">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Map</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextOnly</enum>
     </property>
     <property name="autoRaise">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="labelMap">
     <property name="text">
      <string>Map</string>
     </property>
     <property name="buddy">
      <cstring>btnMapFile</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QComboBox" name="comboBoxColor">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string>Net</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Fontnet</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Night</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QToolButton" name="btnColor">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Color</string>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QPushButton" name="btnFontNet">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Netfont</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QDoubleSpinBox" name="spinZoom">
     <property name="minimum">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QDoubleSpinBox" name="spinX">
     <property name="minimum">
      <double>-99.989999999999995</double>
     </property>
     <property name="maximum">
      <double>99.989999999999995</double>
     </property>
     <property name="singleStep">
      <double>0.050000000000000</double>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QDoubleSpinBox" name="spinY">
     <property name="minimum">
      <double>-99.989999999999995</double>
     </property>
     <property name="maximum">
      <double>99.989999999999995</double>
     </property>
     <property name="singleStep">
      <double>0.050000000000000</double>
     </property>
     <property name="value">
      <double>0.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QCheckBox" name="checkSun">
     <property name="text">
      <string>Sun</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="2">
    <widget class="QCheckBox" name="checkNight">
     <property name="text">
      <string>Night</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="labelZoom">
     <property name="text">
      <string>Zoom</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>spinZoom</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="labelX">
     <property name="text">
      <string>X</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>spinX</cstring>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="labelY">
     <property name="text">
      <string>Y</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>spinY</cstring>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>btnMapFile</tabstop>
  <tabstop>comboBoxColor</tabstop>
  <tabstop>btnColor</tabstop>
  <tabstop>btnFontNet</tabstop>
  <tabstop>spinZoom</tabstop>
  <tabstop>spinX</tabstop>
  <tabstop>spinY</tabstop>
  <tabstop>checkSun</tabstop>
  <tabstop>checkNight</tabstop>
 </tabstops>
 <resources>
  <include location="SatViewer.qrc"/>
 </resources>
 <connections/>
</ui>
